docs/pages/application-access/controls.mdx

```diff
@@ -69,6 +67,67 @@ allow:
     group: "{{external.group}}"
 ```
 
+## Enabling a user to access Azure managed identities
+
+You can authorize your Teleport user to assume an Azure identity and execute
+Azure CLI commands via Teleport. To do so, define a role with the
+`spec.allow.azure_identities` field, as shown below:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: azure-cli-access
+spec:
+  allow:
+    app_labels:
+      '*': '*'
+    azure_identities:
+      - /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/my-resource-group/providers/Microsoft.ManagedIdentity/userAssignedIdentities/teleport-azure
+
+```
+
+This role allows the user to assume the `teleport-azure` identity in order to
+access resources via the Azure CLI.
+
+Another approach is to define a user's permitted Azure identities by configuring
+each user separately. To do so, create a role with one element within
+`azure_identities` set to the template variable `{{internal.azure_identities}}`:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: azure-cli-access
+spec:
+  allow:
+    app_labels:
+      '*': '*'
+    azure_identities:
+      - '{{internal.azure_identities}}'
+```
+
+In this case, when a user authenticates to the Azure CLI via Teleport, the
+Teleport Auth Service populates the `{{internal.azure_identities}}` template
+variable with any Azure identities you have assigned to the user.
+
+To assign Azure identities to a user, run a `tctl users update` command similar
+to the following:
+
+```code
+$ TELEPORT_USER=myuser;
+$ ID1=one_id;
+$ ID2=another_id;
+$ SUBSCRIPTION_ID=00000000-0000-0000-0000-000000000000;
+$ RESOURCE_GROUP="my-resource-group";
```

Comment:
In the command this is referenced as `MY_RESOURCE_GROUP`, not sure which way you want to change it.